.github/pr-title-checker-config.json

```diff
@@ -0,0 +1,17 @@
+{
+  "LABEL": {
+    "name": "title needs formatting",
+    "color": "EEEEEE"
+  },
+  "CHECKS": {
+    "prefixes": ["fix: ", "feat: ", "test: ", "infra: ", "doc: ", "change: "],
```

Comment:
the [contributing guidelines](https://github.com/amazon-braket/amazon-braket-pennylane-plugin-python/blob/main/CONTRIBUTING.md#commit-your-change) also allow:
`breaking: `
`deprecation: `
`depr: `
`feature: `
`documentation: `